doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -0,0 +1,92 @@
+.. _lib_modem_slm:
+
+Modem SLM
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Modem SLM library exposes the AT command interface of the Serial LTE Modem (:ref:`slm_description`) for external devices over a serial interface. This library is for applications running on external MCU, which is built with nRF Connect SDK. And the MCU is connected to nRF9160 SiP through UART.
+
+Functions
+*********
+
+* Manage the serial interface so that the application only decides UART instance to use and configures its DTS.
+* Manage the GPIO pins and support bi-directional indication/wakeup.
+* Send modem or SLM proprietary AT commands and receive response/notification, similar to :ref:`lib_at_host`.
+* Send raw data in SLM data mode.
+* Monitoring AT notification with registered callbacks, similar to :ref:`at_monitor_readme`.
+* Received AT response or notification can be parsed by :ref:`at_cmd_parser_readme`.
```

Comment:
```suggestion

  A received AT response or notification can be parsed by the :ref:`at_cmd_parser_readme` library.
```
Makes this a second line for the previous bullet, since it's not technically something this library does itself, but is related to the previous function.